static/app/routes.tsx

```diff
@@ -2009,6 +2010,7 @@ function buildRoutes(): RouteObject[] {
       children: [
         {
           index: true,
+          handle: {module: ModuleName.MCP},
```

Comment:
note: I think it’s quite weird that these routes are part of the `moduleRoutes` object. That object gets passed to 4 different parent routes: `frontend`, `backend`, `mobile` and `ai` and should probably only contain things that we need everywhere or in multiple places.

But right now, that means I can go to urls like `/insights/frontend/agents/` and it works because the route exists but it renders a non-sensical page: frontend header with the agents page :/